README.md

```diff
@@ -1,8 +1,24 @@
 # Booster Catalog
-Set of known example applications (Boosters) conforming to the minimal set of requirements necessary to be served by launch.openshift.io.
-
-- next: this is the branch from which we'll cut the next tag. Production does not use this branch, it is only used in the staging environment.
-- openshift-online-free; this branch is used in staging AND production. Only the Boosters capable of running in the OSO Free environment are included here.
+Set of known example applications (Boosters) conforming to the minimal set of requirements necessary to be served by developers.redhat.com/launch.
+
+Branching and tagging strategy:
+- `master`:
+    - should always point to the `master` branches of the boosters
+        - if a particular variant of a booster isn't available in a branch, then it should point to the latest tag
+    - can be updated at any time, because it's not used in any Fabric8 Launcher instance we run
+- `next`:
+    - should always point to the latest tags of the boosters
+    - can be updated at any time, because it's only used in the staging Fabric8 Launcher instance we run
+    - booster catalog releases are cut from this branch
```

Comment:
Don't we curt releases from the `free` branch as well?

Reply:
We don't. We can update that at any time.

Reply:
Which seems wrong to my btw. The "free" branch is just as official and public as the tagged release version. Making this updatable makes it possible (even easy) to update a public facing application in production with boosters that have not gone through any testing and Q&A at all.

Reply:
That's true, and would be solved by removing the branch altogether as mentioned in other comments to this PR :-)